pyartcd/pyartcd/pipelines/promote.py

```diff
@@ -1700,40 +1705,62 @@ def _send_release_email(self, release_name: str, advisories: Dict[str, int], jir
             self.runtime.config["email"][f"qe_notification_recipients_ocp{release_version[0]}"],
             subject, content, archive_dir=email_dir, dry_run=self.runtime.dry_run)
 
-    def create_cincinnati_prs(self, assembly_type, release_info):
+    async def create_cincinnati_prs(self, assembly_type, release_info):
         """ Create Cincinnati PRs for the release.
         """
-        if assembly_type == AssemblyTypes.CUSTOM:
-            self._logger.info("Skipping PR creation for custom assembly")
-            return
-
-        if self.skip_cincinnati_prs:
-            self._logger.info("Skipping Cincinnati PRs creation since skip param is set")
-            return
-
-        if self.runtime.dry_run:
-            self._logger.info("[DRY RUN] Would have created Cincinnati PRs.")
-            return
-
         candidate_pr_note = ""
         justifications = release_info["justifications"]
         if justifications:
             candidate_pr_note = "\n".join(justifications)
 
         from_releases = [arch_info["from_release"].split(":")[-1]
                          for arch_info in release_info["content"].values() if "from_release" in arch_info]
-
         advisory_id = 0
         if "advisory" in release_info and release_info["advisory"]:
             advisory_id = release_info["advisory"]
 
-        jenkins.start_cincinnati_prs(
-            from_releases,
-            release_info["name"],
-            advisory_id,
-            candidate_pr_note,
-            self.skip_ota_notification
-        )
+        release_name = release_info["name"]
+        branchName = f"pr-candidate-{release_name}"
```

Comment:
Only cosmetic, but we're using [snake_case](https://en.wikipedia.org/wiki/Snake_case) naming convention everywhere. Why are we introducing [camelCase](https://en.wikipedia.org/wiki/CamelCase) in some places?

Reply:
I just follow the existing name https://github.com/openshift-eng/aos-cd-jobs/blob/master/pipeline-scripts/release.groovy#L943,  I don't have preference

Reply:
Groovy naming is following camelCase, our Python code is trying to align to [PEP8](https://peps.python.org/pep-0008/#function-and-variable-names). If we agree that readability counts, I suggest using a consistent naming, and adopt identifiers such as `branch_name` instead of `branchName`